src/itemLogin/ItemLoginAuthorization.tsx

```diff
@@ -91,12 +90,12 @@ const ItemLoginAuthorization =
       // the item could be fetched without errors
       // because the user is signed in and has access
       // or because the item is public
-      if (item && !item.isEmpty()) {
+      if (item && !item.toSeq().isEmpty()) {
```

Comment:
What about checking the id? Tho I still don't think the item can be empty if it exists.